python/cloudtik/providers/_private/huaweicloud/utils.py

```diff
@@ -234,6 +238,10 @@ def get_default_huaweicloud_cloud_storage(provider_config):
     return cloud_storage_info
 
 
+def get_huaweicloud_workspace_bandwidth_conf():
```

Comment:
Suggest to use workspace config file in provider section to allow user to specify any provider specific options like this.

For example,
provider:
    # bandwidth for public ip in MB
    public_ip_bandwidth: 20

Reply:
I think we need two public ip bandwidth options for workspace NAT and node, apply them when creating workspace NAT and creating head node, I will add them in next commit. Thank you.

Reply:
Agree. Two options look good.